### PR TITLE
Value proposition mapping

### PR DESCRIPTION
**Pull request by ElenaFdR**
Notes from our first user and value proposition mapping session.

### CODE REVIEW

**Comment by nebster** [comment]
i think maybe we should separate these out into one page per user? with maybe the brainstorm info as index?

**Comment by bvhme** [comment]
Great commits @ElenaFdR and great commit messages!

I'd recommend in the future putting a/the short description of your changes (up to 50 chars) in the title bar of the commit, so that they show in the GitHub history. 

Mentioning other contributors and crediting them is awesome, and can always be done in the comments.

See the top one here is collapsed and the bottom one expanded.

Screenshot 2019-04-04 at 09 59 55

File: users/business-management-procurement.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
```
Comment (bvhme) [inline users/business-management-procurement.md]: Remove this white line before the first heading

File: users/vendors-private-sector [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#Vendors and the private sector
```
Comment (bvhme) [inline users/vendors-private-sector]: Missing space between the `#` and the title means the title doesn't get recognised as one

File: users/residents.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+
```
Comment (bvhme) [inline users/residents.md]: I'd remove this empty rule at the beginning of the document (and add one after the title)

**Comment by ElenaFdR** [comment]
Re:
>     * `+` and `-` is mixed for lists (see `users/political.md`) I'd suggest always using `*` but all three are [`compliant`](https://github.github.com/gfm/#list-items)

We've just agreed to always go with asterisks to increase predictability, thereby making it easier for someone else to get started with editing our docs.

**Comment by ElenaFdR** [comment]
@MirjamvT This is ready for your review now.

**Comment by ElenaFdR** [comment]
Outstanding actions:
- find new home for this in About
- @MirjamvT to review

**Comment by ElenaFdR** [comment]
This PR has been replaced by #173. I'm going to delete this branch now. 
(This branch is so snarled that it was faster to rebuild the work in a new branch than fix this branch.)

**Pull request closed without merging**